### PR TITLE
Amazon AWS: Add a command to list SQS Queues

### PR DESCRIPTION
**Pull request by victor-falcon**
## Description

New command to list SQS Queues.

## Type of change

- Bug fix / improvement for my extension

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/amazon-aws/src/queues.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      id={queue}
+      key={queue}
+      title={displayName ?? ""}
+      subtitle={subtitle} 
+      icon="sqs-list-icon.png"
+      accessoryTitle={accessoryTitle}
+      actions={
+        <ActionPanel>
+          <OpenInBrowserAction title="Open in Browser" shortcut={{ modifiers: [], key: "enter" }} url={path} />
+          <CopyToClipboardAction title="Copy path" content={queue} />
```
Comment (pernielsentikaer) [inline extensions/amazon-aws/src/queues.tsx:90]:
```suggestion
          <CopyToClipboardAction title="Copy Path" content={queue} />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @victor-falcon 👋 

A few NITs and comments:

- What if there is 0 items in state.queues, then will the isLoading always be true?  (The same for both script) I think we have to find a better way - do you need help with that?

- I think there should be some graceful error handling when access is denied.

SfrYWLCvNpYArJwf

```
AccessDenied: Access to the resource https://sqs.eu-west-1.amazonaws.com/ is denied.

    resp.error = util.error(new Error(), {
                            ^

Request.extractError:query.js:50:29

---
47:   if (data.Errors) data = data.Errors;
48:   if (data.Error) data = data.Error;
49:   if (data.Code) {
50:     resp.error = util.error(new Error(), {
51:       code: data.Code,
52:       message: data.Message
53:     });
---

Request.callListeners:sequential_executor.js:106:20
Request.emit:sequential_executor.js:78:10
Request.emit:request.js:686:14
Request.transition:request.js:22:10
AcceptorStateMachine.runTo:state_machine.js:14:12
<anonymous>:state_machine.js:26:10
Request.<anonymous>:request.js:38:9
Request.<anonymous>:request.js:688:12
Request.callListeners:sequential_executor.js:116:18
```

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by victor-falcon** [comment]
You're right! I'm going to check this, and I will update this pull request soon. Thanks!

**Comment by victor-falcon** [comment]
@pernielsentikaer I have made the changed requested, but now it's failing in the build because "invalid author", why? 🤔

**Comment by grzegorzkrukowski** [comment]
@pernielsentikaer @victor-falcon the problem with our automatic checks on 'author' has been just fixed and I re-triggered checks for this PR so it should not stop you anymore from merging. Sorry for the inconvenience.

File: extensions/amazon-aws/src/queues.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          }));
+          loadItems(data.QueueUrls);
+        }
+      });
+    }
+    fetch();
+  }, []);
+
+  if (state.hasError) {
+    return (<Detail markdown="We can't find a valid [configuration and credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) in your machine." />)
```
Comment (pernielsentikaer) [inline extensions/amazon-aws/src/queues.tsx:57]:
```suggestion
    return (<Detail markdown="No valid [configuration and credential file] (https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) found in your machine." />)
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Awesome @victor-falcon 💪 

Just a quick NIT before it should be ready to get merged, feel free to use suggested or find your own way to write it 💯 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/amazon-aws/src/queues.tsx (hunk 2 above)
Comment (victor-falcon) [reply extensions/amazon-aws/src/queues.tsx]: Yes, done! 👍

**Comment by pernielsentikaer** [comment]
Great @victor-falcon 💪 

Let get the gem shipped 💎 🚢 

Thanks for the update, much appreciated 💯

**Review by pernielsentikaer — APPROVED** [review]
Great @victor-falcon 💪 

Let get the gem shipped 💎 🚢 

Thanks for the update, much appreciated 💯

**Pull request merged**